File: proyecto6/vehiculo.php
<?php

class Vehiculo {
    public $marca;
    public function encender() {
        return 'El vehiculo está encendido.'."\n";
    }
}

class Coche extends Vehiculo {
    public $modelo;
    public function nombreCoche() {
        echo 'Coche-> Marca: '.$this->marca. ' Modelo: '.$this->modelo;
    }
}

$coche = new Vehiculo;
$coche = new Coche;
echo $coche -> encender();
$coche -> marca = 'Audi';
$coche -> modelo = 'A4';
$coche -> nombreCoche();
?>

File: Java5/Ejercicio_54/miVehiculo/Vehiculo.java
package miVehiculo;

public class Vehiculo {
	
	String marca;
    String modelo;

    public Vehiculo(String marca, String modelo) {
        this.marca = marca;
        this.modelo = modelo;
    }

    public void mostrarDatos() {
        System.out.println("Marca: " + marca + "\nModelo: " + modelo);
    }
}
